support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -32,7 +33,7 @@ object CreateSupportWorkersRequest {
 
   implicit val giftRecipientCodec: Codec[GiftRecipientRequest] = deriveCodec
 
-  implicit val codec: Codec[CreateSupportWorkersRequest] = deriveCodec
+  implicit val codec: Decoder[CreateSupportWorkersRequest] = deriveDecoder[CreateSupportWorkersRequest]
```

Comment:
CreateSupportWorkersRequest is the data that comes from typescript to the backend, so there was no need for an encoder

Reply:
maybe call the val `decoder` rather than `codec`? This is a nitpick though.